crates/builder-api/src/v0_1/builder.rs

```diff
@@ -95,6 +106,7 @@ impl tide_disco::error::Error for Error {
             Error::TxnSubmit { .. } => StatusCode::INTERNAL_SERVER_ERROR,
             Error::Custom { .. } => StatusCode::INTERNAL_SERVER_ERROR,
             Error::BuilderAddress { .. } => StatusCode::INTERNAL_SERVER_ERROR,
+            Error::TxnStatGet { .. } => StatusCode::BAD_REQUEST,
```

Comment:
Nit: Rename this to be consistent with whatever new name we choose for the `get_txn_stat` endpoint.

Reply:
I might still keep it as `TxnStatGet` to keep consistent with `TxnSubmit`.

Reply:
Oh wait...I think it might be better to change to `TxnStat`.